kate_brownbill_dungon_fighters_v1.0.py

```diff
@@ -0,0 +1,838 @@
+    f1b = Fighter([MAGIC_SWNG], "Battle Bot B", "A robot of incapable of using anything outher then magic swing. Try using Firebolt or Magic Reflect", 2, 2, 0)
+    f1c = Fighter([SHIELD_BASH], "Battle Bot C", "A robot of incapable of using anything outher then shield bash. Try using Magic Swing or Forward Swing", 2, 2, 0)
+    f1d = Fighter([MAGIC_RFCT], "Battle Bot D", "A robot of incapable of using anything outher then magic reflect. Try using Forward Swing or Sheild Bash", 2, 2, 0)
+    f1e = Fighter([FIREBOLT], "Battle Bot E", "A robot of incapable of using anything outher then firebolt. Try using Sheild Bash or Magic Reflect", 2, 2, 0)
+    t1 = (p1, [f1a, f1b, f1c, f1d, f1e])
+    
+    #mummy fight
+    p2 = Player(2, "The Mummy Gang", "A shy, pairing of mummys who are doing a favor for the temple guardiens.")
+    f2a = Fighter([FRWD_SWNG, SHIELD_BASH], "Sarah the Mummy", "A mummy deaply in love with David. She's returning a favor to the temple guardiens.", 3, 3, 1)
+    f2b = Fighter([FIREBOLT, MAGIC_RFCT], "David the Mummy", "A mummy deaply in love with Sarah. He's protecting her unconscious body.", 3, 4, 0)
+    t2 = (p2, [f2a, f2b])
+    
+    #the temple guardians
+    p3 = Player(1, "The Temple Guardiens", "A group of robots controlled by ancent spirits.")
+    f3a = Fighter([FRWD_SWNG, MAGIC_SWNG], "Slasher", "The spirit inside this mech loves blades.", 2, 5, 0)
+    f3b = Fighter([FIREBOLT, MAGIC_RFCT], "Sparky", "The spirit inside this mech has studyed magic.", 3, 3, 1)
+    f3c = Fighter([SHIELD_BASH, MAGIC_SWNG], "Scaredy", "The spirit inside this mech is overly protective", 5, 1, 1)
+    t3 = (p3, [f3a, f3b, f3c])
+    
+    #the boss guardian
+    p4 = Player(3, "Program X", "The last defence protecting the temple's tresure.")
+    p4a = Fighter([FRWD_SWNG, SHIELD_BASH, MAGIC_SWNG], "The Guardian", "The ultimate wepon used by program X.", 4 ,3 ,3)
+    t4 = (p4, [p4a])
+    
+    #area name, area text, and new fighter
+    area_name = "Temple"
+    begin_text = "You have gathered your best adventurers, ahead of you is a temple that holds a specacular tresure. Holding your head up high, you charge the temple."
+    end_text = "As you entered the inner temple, a tall necromancer clad in dark robes turned around to face you. In his hands was the very tresure you were seeking. Smiling, he desends into the caverns below."
+    new_fighter_txt = "Behind you, a fluffy air elementle with a bow on her head floats over to you. 'You going after that necromancer? You might need a hand down there! I would be happy to help!'"
+    af = Fighter([FIREBOLT, MAGIC_RFCT, MAGIC_SWNG], "Puff", "An air elementle made of clouds. She's an expert at magic.", 4, 5, 1)
+    
+    #putting togeather everything
+    area_info = (area_name, begin_text, end_text, new_fighter_txt, af, [t1, t2, t3, t4])
```

Comment:
I think this is something that could be much more readable as a class.

You have to kind of know that area_name is the first thing in the return list.  I see code above like...

`        for t in a[5]:`

and it's hard to reason about.  "oh, it's [t1, t2, t3, t4 ,t5] here,  but what's that...
t2 is (p2, [f2a, f2b])

Ah,  p2 is the opposing player and f2a and f2b are the monsters in the party.

if we returned a

`class AreaInfo`

And that class had a method that said...

`area_info.get_list_of_opposing_parties()`

Then

`        for t in a[5]:`

becomes

`          for party in area.get_list_of_opposing_parties():`